tmt/export.py

```diff
@@ -46,21 +47,25 @@ def export_to_nitrate(test, create, general):
     try:
         nitrate_id = test.node.get('extra-nitrate')[3:]
         nitrate_case = nitrate.TestCase(int(nitrate_id))
-        nitrate_case.summary # Make sure we connect to the server now
+        nitrate_case.summary  # Make sure we connect to the server now
         echo(style(f"Test case '{nitrate_case.identifier}' found.", fg='blue'))
     except TypeError:
         # Create a new nitrate test case
         if create:
             nitrate_case = create_nitrate_case(test)
             new_test_created = True
+            test._metadata['extra-summary'] = nitrate_case.summary
```

Comment:
Could you add a short comment explaining why we are storing the extra summary from nitrate summary here?

Reply:
I added comment line in 336dab7 hopefully its what you meant